data/json/items/ammo/8x40mm.json

```diff
@@ -12,7 +12,7 @@
   {
     "id": "8mm_caseless",
     "type": "AMMO",
-    "name": { "str": "8x40mm caseless" },
+    "name": { "str": "8x40mm caseless", "str_pl": "8x40mm caselesses" },
```

Comment:
I am not sure this one is correct.

Reply:
What's your suggestion?

Reply:
i suggest that the plural form is also 8x40mm caseless

Reply:
Done.